### PR TITLE
Update bhbotlist.tech to add features

### PR DESCRIPTION
**Pull request by anshtyagi0**
Updating bhbotlist features

### CODE REVIEW

**Comment by anshtyagi0** [comment]
Someone please merge it.

File: data/lists/bhbotlist.tech.json [hunk 1]
```diff
@@ -43,7 +43,11 @@
         "html-long-description",
         "iframe-long-description",
         "offers-paid-promotion",
+        "has-mobile-support",
         "requires-owner-in-server",
-        "server-count-api"
+        "server-count-api",
+        "votes-sent-to-webhooks",
+        "vanity-urls-for-all",
```
Comment (MattIPv4) [inline data/lists/bhbotlist.tech.json:50]: I'm not seeing vanity URLs on your site? Everything still seems to use bot IDs?

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: data/lists/bhbotlist.tech.json (hunk 1 above)
Comment (anshtyagi0) [reply data/lists/bhbotlist.tech.json:50]: We use it for certified bots like: https://bhbotlist.tech/stella

Comment (anshtyagi0) [reply data/lists/bhbotlist.tech.json:50]: we don't change existing url

Comment (MattIPv4) [reply data/lists/bhbotlist.tech.json:50]: Okay, so if its just for certified bots, then it isn't for all bots as the feature says is it?

Comment (MattIPv4) [reply data/lists/bhbotlist.tech.json:50]: There is a separate feature if certified bots get vanity URLs, not all bots: https://github.com/botblock/data/blob/master/data/features/certified-bot-vanity-urls.json

**Comment by MattIPv4** [comment]
No progress, closing.

**Pull request closed without merging**